src/Widgets/DataLabelWidget/DataLabelWidgetComponent.tsx

```diff
@@ -84,3 +85,13 @@ function Datetime({ value }: { value: unknown }) {
 
   return <RelativeDate date={date} />
 }
+
+function URL({ value }: { value: unknown }) {
```

Comment:
* [ ] FYI For suffixes we use camel instead of all uppercase: `*Id` not `*ID`, `*Url` not `*URL`. Thus `Url` would be the more consistent casing here.